<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schedule</class>
 <widget class="QWidget" name="Schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_Days">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Day:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Days">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sunday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thursday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Friday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saturday</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_Time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Subject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Subject">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Load">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
